pom.xml

```diff
@@ -6,7 +6,7 @@
 
   <groupId>gov.cms.madie</groupId>
   <artifactId>madie-java-models</artifactId>
-  <version>0.3.9-SNAPSHOT</version>
+  <version>0.3.9.1-SNAPSHOT</version>
```

Comment:
Not sure if you have a special intention with that numbering... but using 0.3.10-SNAPSHOT is probably better.

Reply:
@gregory-akins Ok thanks will do.

Reply:
Changed to 0.3.10-SNAPSHOT so resolving this conversation.